contrib/init/upstart/docker.conf

```
@@ -3,7 +3,7 @@ description "Docker daemon"
 start on (filesystem and net-device-up IFACE!=lo)
 stop on runlevel [!2345]
 
-limit nofile 524288 1048576
+limit nofile 1024 524288
```

Reverting as config is being dropped.

```suggestion
limit nofile 524288 1048576
```